android/res/layout-land/routing_details.xml

```diff
@@ -9,10 +9,10 @@
   <TextView
     android:id="@+id/time"
     android:singleLine="true"
-    android:ellipsize="middle"
+    android:ellipsize="end"
```

Comment:
Should we remove it completely?

Reply:
Its needed for some cases still...

![OM-route-distance-long-2lines-ellipsize](https://user-images.githubusercontent.com/18434508/147220558-78d6c99f-df00-4cad-9cf8-6c9f6c1a85b1.png)

Reply:
BTW we can win some more space by removing whitespace between numbers and corresponding units,
e.g.
`11h 55min - 153km` - at the expense of slightly less readability

Reply:
Let's try and see how it looks in normal and extreme cases.

Reply:
### Large font:

![nospace_1](https://user-images.githubusercontent.com/18434508/147254203-1d4aecad-a9b2-4691-8918-c18db7600a4d.png)
![nospace_2](https://user-images.githubusercontent.com/18434508/147254217-c02a6256-b8c0-433f-81d1-5637808cd4a9.png)
![nospace_3](https://user-images.githubusercontent.com/18434508/147254223-a0d0bfa3-384f-48ed-8b67-0aabe7d3fb68.png)
![nospace_4](https://user-images.githubusercontent.com/18434508/147254231-1847ebf9-18f8-41a3-8e03-b6affd06a072.png)

### Normal font:
![nospace_sm_1](https://user-images.githubusercontent.com/18434508/147254295-ec7e46f8-6ad5-450f-baaf-53c7cae38140.png)
![nospace_sm_2](https://user-images.githubusercontent.com/18434508/147254301-8dfaedba-e838-44a9-ba0b-d60f54fcc3d0.png)

### It also affects display in navigation mode:
![nospace_nav](https://user-images.githubusercontent.com/18434508/147254369-3dba242a-9b86-4404-835a-562f6b02278d.png)

I don't see any improvement TBH, so I suggest leave it as it was with whitespace before units (the current commit in the PR).

Reply:
Ok, thanks. Let's make it ideal later )